python/ml4ir/applications/classification/tests/test_classification_model.py

```diff
@@ -42,7 +42,7 @@ def test_group_metrics_df(self):
         Test the dimensions of the grouped metrics
         """
         # Metrics cardinality and names
-        metrics = self.classification_pipeline.metrics_keys  # Metrics during training
+        metrics = ["loss"] + self.classification_pipeline.metrics_keys  # Metrics during training
```

Comment:
Did you get a chance to train classification models with these changes @lastmansleeping ? Should we have a follow up item to ensure end-to-end flow works as expected?

Reply:
I have verified classification models too. You should be able to verify with
```
python ml4ir/applications/classification/pipeline.py \
--data_dir ml4ir/applications/classification/tests/data/tfrecord \
--feature_config ml4ir/applications/classification/tests/data/configs/feature_config.yaml \
--model_config ml4ir/applications/classification/tests/data/configs/model_config.yaml \
--batch_size 32 \
--run_id test \
--data_format tfrecord \
--execution_mode train_evaluate \
--num_epochs 2
```

Please note that the last step of this will fail with the default model_config because temperature scaling is not yet supported.

All other classification, ranking and base tests are passing though.